app/main.py

```diff
@@ -1,8 +1,45 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        if name in Person.people:
```

Comment:
The check for an existing person with the same name is correct, but it only prints an error message. Consider raising an exception or handling this case more robustly if needed.